web-registry/src/components/organisms/ProjectMiddleSection/ProjectMiddleSection.tsx

```diff
@@ -0,0 +1,60 @@
+import { Box, Grid } from '@mui/material';
+
+import Section from 'web-components/lib/components/section';
+import { Title } from 'web-components/lib/components/typography';
+
+import { ProjectBatchTotals } from '../../molecules';
+import { CreditBatches } from '../CreditBatches/CreditBatches';
+import { useProjectTopSectionStyles } from './ProjectMiddleSection.styles';
+import { ProjectMiddleSectionProps } from './ProjectMiddleSection.types';
+
+function ProjectMiddleSection({
+  offChainProject,
+  batchData,
+  paginationParams,
+  setPaginationParams,
+  projectWithOrderData,
+  soldOutProjectsIds,
+}: ProjectMiddleSectionProps): JSX.Element {
+  const { classes } = useProjectTopSectionStyles();
+
+  return (
+    <Section classes={{ root: classes.section }}>
+      <Grid container>
+        <Grid item xs={12} md={8} sx={{ pr: { md: 19 } }}>
+          {batchData?.totals && (
+            <ProjectBatchTotals
+              projectWithOrderData={projectWithOrderData}
+              soldOutProjectsIds={soldOutProjectsIds}
+              totals={batchData.totals}
+              sx={{
+                mt: { xs: 10, sm: 12, md: 16 },
+                mb: { xs: 10, sm: 12, md: 0 },
+              }}
+            />
+          )}
```

Comment:
shouldn't this be up the gallery photo and additional info section (looking at the design: https://www.figma.com/file/seUiQM2SPt78KJiDL0Ea8m/Basic-Ecocredits-Ledger-View-%2F-Marketplace?node-id=6668%3A173353&t=VruhIAw1fabahQwz-1)